### PR TITLE
docs(pt): add `_collections` translated

### PR DESCRIPTION
**Pull request by nazarepiedady**
In this pull request, I am sending the `_collections` translated.

I remember that one day @smarroufin said that have the whole
`pt/docs/` folder translated would be enough to generate a new
website build putting the Portuguese version of the documentation
available, but I would like to ask you for a little 🙏 patience because
I am almost finished translating all files in the `pt/` folder 😊.

With this pull request, it will only be missing the following folders:

- `pt/announcements/`: it being translated.
- `pt/case-studies/`: it being translated.
- `pt/contribution-guide/`: the only file there is already translated.
- `pt/tutorials`: the translation has not started yet.

One more thing 😊,
@smarroufin what kind of contribution someone should do to be here [nuxt.org/teams](https://nuxtjs.org/teams) in the community section?

@smarroufin and @ChristopheCVB it is your turn.

### CODE REVIEW

**Review by ChristopheCVB — CHANGES_REQUESTED** [review]
Just a little typo, thanks for your hard work!

**Comment by smarroufin** [comment]
Hey @nazarepiedady thanks again for all the work! :green_heart: 

:warning: before you go furthermore into your translations, `announcements/` and `case-studies/` are not meant to be translated. See them as blog articles. Sorry, this is not obvious and we will eventually do something to warn contributors about that.

`_collections` will be enough to enable `pt` language, simplifying reviews with the deploy preview. And you will still be able to translate `contribution-guide/` and `tutorials/` freely.

I'm gonna see about the contributors page and will come back to you :slightly_smiling_face:

File: content/pt/_collections/teams/2.community.md [hunk 1]
```diff
@@ -237,7 +237,7 @@ members:
       - gitHub: 'https://github.com/kevinmarrec'
       - twitter: 'https://twitter.com/K_Marrec'
       - website: 'https://marrec.io/'
-  -
+  -
```
Comment (smarroufin) [inline content/pt/_collections/teams/2.community.md:240]: You can revert this file, no need to change this.

File: content/pt/_collections/testimonials/index.md [hunk 2]
```diff
@@ -1,34 +1,34 @@
 ---
 testimonials:
   -
-    testimonial: 'Nuxt offers a compelling solution and a great ecosystem to help you ship fullstack Vue apps that are performant and SEO friendly. The flexibility to choose between SSR and SSG is icing on the cake.'
+    testimonial: 'A Nuxt oferece uma solução convincente e um excelente ecossistema para ajudar você a entregar aplicações Vue completas que são funcionais e amigáveis ao SEO. A flexibilidade para escolher entre a Renderização no Lado do Servidor (SSR, sigla em Inglês) e Geração de Sítios Estáticos (SSG, sigla em Inglês) é a cereja sobre o bolo.'
```
Comment (smarroufin) [inline content/pt/_collections/testimonials/index.md:4]: Testimonials should not be translated. Those sentences are quotes of their author and must not be translated or altered in any ways.

**Review by smarroufin — CHANGES_REQUESTED** [review]

**Comment by smarroufin** [comment]
@nazarepiedady I discussed the contributors page update with some people. This page will disappear very soon, so we won't update it to delete it right after.

**Review by ChristopheCVB — APPROVED** [review]

**Pull request merged**